### PR TITLE
add about page

### PR DESCRIPTION
**Pull request by jonkiky**
Except the Model Page, Discussion is needed with Phil to confirm the style change.

### CODE REVIEW

File: src/pages/about/aboutView.js [hunk 1]
```diff
@@ -1,24 +1,24 @@
 import React from 'react';
-import { AboutHeader, AboutBody } from 'bento-components';
+import { AboutBody } from 'bento-components';
 import { withStyles } from '@material-ui/core';
-import Stats from '../../components/Stats/AllStatsController';
+import AboutHeader from './aboutHeader';
```
Comment (iksheth) [inline src/pages/about/aboutView.js:4]: @jonkiky Should we import this new custom header to bento-tools ?

**Review by iksheth — APPROVED** [review]
LGTM 👍🏻

File: src/pages/about/aboutView.js (hunk 1 above)
Comment (jonkiky) [reply src/pages/about/aboutView.js:4]: I thought about this. But this Header does not have any functions, only the ICDC specific style. 
Less reusable in other data commons.  So put it as optional.

**Comment by skiran86** [comment]
@jonkiky Conflicts

**Review by skiran86 — APPROVED** [review]

**Review by AjayDoddapaneni — APPROVED** [review]

**Pull request merged**